moveit_core/robot_model/src/revolute_joint_model.cpp

```diff
@@ -168,11 +168,12 @@ double RevoluteJointModel::distance(const double* values1, const double* values2
 
 bool RevoluteJointModel::satisfiesPositionBounds(const double* values, const Bounds& bounds, double margin) const
 {
-  if (!continuous_)
+  if (continuous_)
```

Comment:
@dcconner Thanks for considering my comment.
Here, the curly brackets could/should be dropped. Otherwise clang enforces very verbose code...

Reply:
OK.  I see the clang issue.  I'm going to rebase again, clean up, push to verify that everything passes, and then re-do this as a single squashed commit and do a force push to avoid the commit spam.

Reply:
Thanks for taking care of a clean commit history. In this case, however, that's not necessary as github allows us to squash-merge everything on top of the current HEAD anyway.